.github/workflows/appsec.yml

```diff
@@ -70,6 +70,24 @@ jobs:
           cd ./v2/contrib/gin-gonic/gin
           env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.gin.xml -- -v .
           cd -
+          cd ./v2/contrib/google.golang.org/grpc
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.grpc.xml -- -v .
+          cd -
+          cd ./v2/contrib/net/http
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.http.xml -- -v .
+          cd -
+          cd ./v2/contrib/gorilla/mux
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.gorilla.xml -- -v .
+          cd -
+          cd ./v2/contrib/go-chi/chi
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.chi.xml -- -v .
+          cd -
+          cd ./v2/contrib/go-chi/chi.v5
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.chi.xml -- -v .
+          cd -
+          cd ./v2/contrib/labstack/echo.v4
+          env ${{ matrix.cgocheck }} CGO_ENABLED=${{ matrix.cgo_enabled }} ${{ matrix.appsec_enabled }} gotestsum --junitfile $JUNIT_REPORT.echo.xml -- -v .
+          cd -
```

Comment:
Not proud of this but it gets the work done. Will improve it when merging into main.